### DISCUSSION TITLE
Why is the text in my .nav-item not vertically centered and how can I fix it?

### DISCUSSION THREAD

**Discussion by danielniccoli** [twbs/bootstrap#37878] [Q&A] [answered]
Why is the text in my .nav-item not vertically centered and how can I fix it?

![grafik](https://user-images.githubusercontent.com/2971735/212475331-ae22dd4e-d91c-45b1-9e52-1e6e098101f6.png)

https://codepen.io/vermilioncore/pen/zYLzQRr?editors=1000

**Comment by mdo** [comment, marked as answer]
Windows' font sits lower unfortunately than Mac and Linux. You're removing all the padding that helps balance this out some as well, so it's more noticeable. Not much we can do about it I'm afraid.
